internal/support/notifications/errors/types.go

```diff
@@ -39,3 +39,15 @@ func (e ErrSubscriptionNotFound) Error() string {
 func NewErrSubscriptionNotFound(slug string) error {
 	return ErrSubscriptionNotFound{slug: slug}
 }
+
+type ErrInvalidEmailAddresses struct {
```

Comment:
Thank you very much, I can see you had suggested this in one of your yesterday's comment, but I managed to miss and jumped to the conclusion that I had thought about earlier. Sorry about that.
This helps keep both the parts clean and easy to follow, instead of 'either-or' situation it was evolving into. Made the changes.